viewer_clients/web-client/style/codecheckerviewer.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway|Source+Code+Pro');
```

Comment:
I like new fonts. But after building a package we should not depend on external web resources. I see there is a fall back font, but I'm not a big fan of this.